docs/openapi.json

```diff
@@ -455,6 +455,17 @@
                   },
                   "type": "object"
                 },
+                "maximum_interactive_session_inactivity_period": {
```

Comment:
Since you modified the OpenAPI specification you have to call the `generate_openapi_spec.py` script to publish these changes to `reana-commons` (and create another PR there)